pkg/containerspec/containerspec.go

```diff
@@ -57,46 +59,68 @@ func (spec *PaastaContainerSpec) GetContainerResources() (*corev1.ResourceRequir
 		memory = defaultMemory
 	}
 	if _, err := strconv.Atoi(string(memory)); err == nil {
-		// memory value looks like a number, let's treat it as MB according to PaaSTA default
-		memory = memory + "M"
+		// value looks like a number, let's treat it as MB according to PaaSTA default
+		memory = memory + "Mi"
```

Comment:
PaaSTA uses `M` for memory values without suffixes, we better keep that for the backward compatibility.

Reply:
What's the expected meaning of this "M" suffix? in K8s resource stanzas that means 1e6 instead of 2^20 and we copy this suffix straight into K8s requirements stanzas.

Reply:
1000000 bytes

Reply:
We discussed this offline and decided to go with `Mi` (aka `2^20`) as the implicit suffix to numbers without suffixes.